### ISSUE TITLE
[resign] Enable Nested App bundle_id Update

### ISSUE THREAD

**Issue by mikerunnals** [fastlane/fastlane#9294]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

Update the Resign Action to accept a Hash of nested app names (APP_PATH basename) to new bundle ids to allow updating all of the bundle ids within an app. This would work similarly to the current support for multiple provisioning profiles and support for the current bundle_id String would be maintained.

``` 
bundle_id_hash = {
  "My App Name" => "at.fastlane",
  "My App Ext Name" => "at.fastlane.today"
}
 ```

```
resign(
  ipa: newipa_path,
  signing_identity: identity,
  bundle_id: bundle_id_hash
  provisioning_profile: profiles_hash
)
```

**Comment by mikerunnals** [comment]
This functionality is already supported by resign shell script. Feedback comments on the PR.

tl;dr The bundle ids keys specified in the profiles argument hash should be the existing/old bundle ids in the current ipa. Resign set the new bundle ids to those specified within the new mobileprovision files.
